### ISSUE TITLE
Exception thrown in clean-css 3.2.5

### ISSUE THREAD

**Issue by hostep** [jakubpawlowicz/clean-css#550]
I get this after upgrading from 3.1.9 to 3.2.5

```
path.js:488
      throw new TypeError('Arguments to path.join must be strings');
      ^

TypeError: Arguments to path.join must be strings
    at Object.posix.join (path.js:488:13)
    at originalPositionIn (node_modules/clean-css/lib/utils/input-source-map-tracker.js:152:53)
    at InputSourceMapStore.originalPositionFor (node_modules/clean-css/lib/utils/input-source-map-tracker.js:240:10)
    at snapshot (node_modules/clean-css/lib/utils/source-maps.js:54:77)
    at track (node_modules/clean-css/lib/utils/source-maps.js:110:5)
    at Object.context.track (node_modules/clean-css/lib/selectors/tokenizer.js:28:63)
    at Object.Extractors.properties (node_modules/clean-css/lib/utils/extractors.js:88:46)
    at tokenize (node_modules/clean-css/lib/selectors/tokenizer.js:221:26)
    at Tokenizer.toTokens (node_modules/clean-css/lib/selectors/tokenizer.js:40:10)
    at SelectorsOptimizer.process (node_modules/clean-css/lib/selectors/optimizer.js:12:68)
```

If you need more info, let me know, I haven't looked at it in detail yet.

**Comment by hostep** [comment]
Small update, the exception isn't thrown in 3.2.4, so it has to be a change in 3.2.5

And this is the command I try to run:
`cleancss --output style.css --s0 --source-map style.css`

**Comment by hostep** [comment]
Another update: I tracked it down to the bootstrap library, if you try to run clean-css on bootstrap 3.3.4, it throws this error, so now you have a test case :)

`cleancss --output ~/Downloads/bootstrap-3.3.4-dist/css/bootstrap-clean.css --s0 --source-map ~/Downloads/bootstrap-3.3.4-dist/css/bootstrap.css`

**Comment by jakubpawlowicz** [comment]
Thanks, I'll take a look shortly!

**Comment by jakubpawlowicz** [comment]
Thanks for the test case, I can see it failing now. It actually since 3.2.0 there not 3.2.4, but that does not matter. Fix is coming!

**Comment by jakubpawlowicz** [comment]
The fix is pretty simple, but when searching for it I found out that bootstrap mapping is pretty much broken because of `sourcesContent` data. I need to figure out why first.

**Comment by hostep** [comment]
Ok, no problem, take your time, we will stay at 3.1.9 for the time being :)

**Comment by jakubpawlowicz** [comment]
Fixed in 3.2.6 although I opened #557 to continue investigating the wrong line tracking. Stay tuned.

**Comment by hostep** [comment]
Ok, crash is gone now, thanks!
